### PR TITLE
Api 1202 java 14 upgrade mock ee

### PR DESCRIPTION
**Pull request by taylorrassmannlibertyits**

### CODE REVIEW

File: mock-ee-tests/pom.xml [hunk 1]
```diff
@@ -4,7 +4,7 @@
   <parent>
     <groupId>gov.va.api.health</groupId>
     <artifactId>health-apis-parent</artifactId>
-    <version>6.0.7</version>
+    <version>7.0.0</version>
     <relativePath/>
   </parent>
   <artifactId>mock-ee-tests</artifactId>
```
Comment (jhulbert-va) [inline mock-ee-tests/pom.xml:10]: Upgrade to the next major version.

File: mock-ee/pom.xml [hunk 2]
```diff
@@ -17,6 +17,12 @@
     <jacoco.coverage>0.94</jacoco.coverage>
   </properties>
   <dependencies>
+    <!-- https://mvnrepository.com/artifact/commons-io/commons-io -->
```
Comment (jhulbert-va) [inline mock-ee/pom.xml:20]: You can remove this comment.

File: pom.xml [hunk 3]
```diff
@@ -4,7 +4,7 @@
   <parent>
     <groupId>gov.va.api.health</groupId>
     <artifactId>health-apis-parent</artifactId>
-    <version>6.0.7</version>
+    <version>7.0.0</version>
   </parent>
   <artifactId>mock-ee-parent</artifactId>
   <version>2.0.29-SNAPSHOT</version>
```
Comment (jhulbert-va) [inline pom.xml:10]: Update this to the next major version.

File: mock-ee/pom.xml [hunk 4]
```diff
@@ -17,6 +17,11 @@
     <jacoco.coverage>0.94</jacoco.coverage>
   </properties>
   <dependencies>
+    <dependency>
+      <groupId>commons-io</groupId>
```
Comment (jblefkove-va) [inline mock-ee/pom.xml:21]: Since this is only used in a couple places, I'd like to err  on the side of just refactoring it out.

Comment (taylorrassmannlibertyits) [reply mock-ee/pom.xml]: Cool, thanks for taking it out for me! :)

**Pull request merged**